normcap: 0.4.4 -> 0.5.4

pbsds opened the pull request:
## Description of changes

Diff: https://github.com/dynobo/normcap/compare/refs/tags/v0.4.4...v0.5.4


* closes #278448

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

pbsds committed:
normcap: 0.4.4 -> 0.5.2

Diff: https://github.com/dynobo/normcap/compare/refs/tags/v0.4.4...v0.5.2

cafkafk commented:
Result of `nixpkgs-review pr 275063` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>normcap</li>
    <li>normcap.dist</li>
  </ul>
</details>

cafkafk reviewed (changes requested):
fails to build, seems to be something with tests

pbsds commented:
Unable to reproduce on x86_64-linux on master. :S

But the darwin failures are a considering they aren't broken on master

* https://hydra.nixos.org/build/244385402
* https://hydra.nixos.org/build/244383720

I'm at a loss on what to do

nadir-ishiguro commented:
Just tried it and it builds and runs fine both on master ( 74b1db5164643b0165a9ad32a372d581c6ce9a80 ) and on nixos-unstable for me.

My machines:

```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.8, NixOS, 24.05 (Uakari), 24.05.20231222.6df37dc`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/55ql4j8d47xjvfa8xggjddgfwny4n9j7-source`
```

and

```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.8-2-cachyos, EndeavourOS, noversion, 2023.03.06`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - nixpkgs: `/nix/store/61gw1axgxrvx0bnxv2dla6zhkgwv45zr-source`

```

nadir-ishiguro commented:
Well, that's weird. On *another* machine I just got this build error and after checking it now doesn't build on the aforementioned machines anymore: 

```
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/8085wav5zkhj8zi4jnk7xakhpf57nncf-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tests/tests_screengrab/test_utils.py
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
[1m* Getting build dependencies for wheel...[0m
[1m* Building wheel...[0m
68 of 68 messages (100%) translated in /build/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.mo
68 of 68 messages (100%) translated in /build/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /build/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.po
compiling catalog /build/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.po to /build/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.mo
[1m[92mSuccessfully built [4mnormcap-0.5.2-py3-none-any.whl[0m[1m[92m[0m
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/normcap-0.5.2...OK
Repacking wheel as ./normcap-0.5.2-py3-none-any.whl...creating './normcap-0.5.2-py3-none-any.whl' and adding 'unpacked/normcap-0.5.2' to it
adding 'normcap/__init__.py'
adding 'normcap/__main__.py'
adding 'normcap/app.py'
adding 'normcap/utils.py'
adding 'normcap/clipboard/__init__.py'
adding 'normcap/clipboard/main.py'
adding 'normcap/clipboard/handlers/__init__.py'
adding 'normcap/clipboard/handlers/base.py'
adding 'normcap/clipboard/handlers/pbcopy.py'
adding 'normcap/clipboard/handlers/qtclipboard.py'
adding 'normcap/clipboard/handlers/windll.py'
adding 'normcap/clipboard/handlers/wlclipboard.py'
adding 'normcap/clipboard/handlers/xclip.py'
adding 'normcap/gui/__init__.py'
adding 'normcap/gui/constants.py'
adding 'normcap/gui/downloader.py'
adding 'normcap/gui/introduction.py'
adding 'normcap/gui/language_manager.py'
adding 'normcap/gui/loading_indicator.py'
adding 'normcap/gui/localization.py'
adding 'normcap/gui/menu_button.py'
adding 'normcap/gui/models.py'
adding 'normcap/gui/notification.py'
adding 'normcap/gui/resources.py'
adding 'normcap/gui/settings.py'
adding 'normcap/gui/system_info.py'
adding 'normcap/gui/tray.py'
adding 'normcap/gui/update_check.py'
adding 'normcap/gui/utils.py'
adding 'normcap/gui/window.py'
adding 'normcap/ocr/__init__.py'
adding 'normcap/ocr/enhance.py'
adding 'normcap/ocr/magic.py'
adding 'normcap/ocr/models.py'
adding 'normcap/ocr/recognize.py'
adding 'normcap/ocr/tesseract.py'
adding 'normcap/ocr/magics/__init__.py'
adding 'normcap/ocr/magics/base_magic.py'
adding 'normcap/ocr/magics/email_magic.py'
adding 'normcap/ocr/magics/multi_line_magic.py'
adding 'normcap/ocr/magics/paragraph_magic.py'
adding 'normcap/ocr/magics/single_line_magic.py'
adding 'normcap/ocr/magics/url_magic.py'
adding 'normcap/ocr/magics/url_utils.py'
adding 'normcap/resources/README.md'
adding 'normcap/resources/__init__.py'
adding 'normcap/resources/icons/normcap.png'
adding 'normcap/resources/icons/notification.png'
adding 'normcap/resources/icons/parse.png'
adding 'normcap/resources/icons/raw.png'
adding 'normcap/resources/icons/resources.qrc'
adding 'normcap/resources/icons/settings.png'
adding 'normcap/resources/icons/tray.png'
adding 'normcap/resources/icons/tray_done.png'
adding 'normcap/resources/icons/tray_processing.png'
adding 'normcap/resources/images/darwin-intro-1.png'
adding 'normcap/resources/images/darwin-intro-2.png'
adding 'normcap/resources/images/darwin-intro-3.png'
adding 'normcap/resources/images/darwin-intro-4.png'
adding 'normcap/resources/images/linux-intro-1.png'
adding 'normcap/resources/images/linux-intro-2.png'
adding 'normcap/resources/images/linux-intro-3.png'
adding 'normcap/resources/images/linux-intro-4.png'
adding 'normcap/resources/images/win32-intro-1.png'
adding 'normcap/resources/images/win32-intro-2.png'
adding 'normcap/resources/images/win32-intro-3.png'
adding 'normcap/resources/images/win32-intro-4.png'
adding 'normcap/resources/locales/README.md'
adding 'normcap/resources/locales/de_DE/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/es_ES/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/fr_FR/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/hi_IN/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/it_IT/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/pl_PL/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/pt_PT/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/ru_RU/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/sv_SE/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/uk_UA/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/zh_CN/LC_MESSAGES/messages.mo'
adding 'normcap/resources/tessdata/.keep'
adding 'normcap/resources/tessdata/LICENSE.txt'
adding 'normcap/resources/tessdata/README.txt'
adding 'normcap/screengrab/__init__.py'
adding 'normcap/screengrab/dbus_portal.py'
adding 'normcap/screengrab/dbus_shell.py'
adding 'normcap/screengrab/exceptions.py'
adding 'normcap/screengrab/grim.py'
adding 'normcap/screengrab/main.py'
adding 'normcap/screengrab/permissions.py'
adding 'normcap/screengrab/qt.py'
adding 'normcap/screengrab/utils.py'
adding 'normcap-0.5.2.dist-info/licenses/LICENSE'
adding 'normcap-0.5.2.dist-info/METADATA'
adding 'normcap-0.5.2.dist-info/WHEEL'
adding 'normcap-0.5.2.dist-info/entry_points.txt'
adding 'normcap-0.5.2.dist-info/RECORD'
OK
/build/source
Finished executing pypaBuildPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "installPhase" }
Running phase: installPhase
Executing pypaInstallPhase
Successfully installed normcap-0.5.2-py3-none-any.whl
Finished executing pypaInstallPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
Running phase: pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
Running phase: fixupPhase
shrinking RPATHs of ELF executables and libraries in /nix/store/j4a0ynm3l36sbzgfffzbd55bf00rva8g-normcap-0.5.2
checking for references to /build/ in /nix/store/j4a0ynm3l36sbzgfffzbd55bf00rva8g-normcap-0.5.2...
patching script interpreter paths in /nix/store/j4a0ynm3l36sbzgfffzbd55bf00rva8g-normcap-0.5.2
stripping (with command strip and flags -S -p) in  /nix/store/j4a0ynm3l36sbzgfffzbd55bf00rva8g-normcap-0.5.2/lib /nix/store/j4a0ynm3l36sbzgfffzbd55bf00rva8g-normcap-0.5.2/bin
shrinking RPATHs of ELF executables and libraries in /nix/store/5daw29ha297rd6pl1xbsiq7l62w1scc1-normcap-0.5.2-dist
checking for references to /build/ in /nix/store/5daw29ha297rd6pl1xbsiq7l62w1scc1-normcap-0.5.2-dist...
patching script interpreter paths in /nix/store/5daw29ha297rd6pl1xbsiq7l62w1scc1-normcap-0.5.2-dist
Rewriting #!/nix/store/5k91mg4qjylxbfvrv748smfh51ppjq0g-python3-3.11.6/bin/python3.11 to #!/nix/store/5k91mg4qjylxbfvrv748smfh51ppjq0g-python3-3.11.6
wrapping `/nix/store/j4a0ynm3l36sbzgfffzbd55bf00rva8g-normcap-0.5.2/bin/normcap'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
@nix { "action": "setPhase", "phase": "installCheckPhase" }
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Running phase: pythonImportsCheckPhase
Executing pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Running phase: pytestCheckPhase
Executing pytestCheckPhase
_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.
The XKEYBOARD keymap compiler (xkbcomp) reports:
> Warning:          Could not resolve keysym XF86CameraAccessEnable
> Warning:          Could not resolve keysym XF86CameraAccessDisable
> Warning:          Could not resolve keysym XF86CameraAccessToggle
> Warning:          Could not resolve keysym XF86NextElement
> Warning:          Could not resolve keysym XF86PreviousElement
> Warning:          Could not resolve keysym XF86AutopilotEngageToggle
> Warning:          Could not resolve keysym XF86MarkWaypoint
> Warning:          Could not resolve keysym XF86Sos
> Warning:          Could not resolve keysym XF86NavChart
> Warning:          Could not resolve keysym XF86FishingChart
> Warning:          Could not resolve keysym XF86SingleRangeRadar
> Warning:          Could not resolve keysym XF86DualRangeRadar
> Warning:          Could not resolve keysym XF86RadarOverlay
> Warning:          Could not resolve keysym XF86TraditionalSonar
> Warning:          Could not resolve keysym XF86ClearvuSonar
> Warning:          Could not resolve keysym XF86SidevuSonar
> Warning:          Could not resolve keysym XF86NavInfo
Errors from xkbcomp are not fatal to the X server
[1m============================= test session starts ==============================[0m
platform linux -- Python 3.11.6, pytest-7.4.3, pluggy-1.3.0
PySide6 6.6.0 -- Qt runtime 6.6.1 -- Qt compiled 6.6.1
rootdir: /build/source
configfile: pyproject.toml
testpaths: tests
plugins: qt-4.2.0, xvfb-3.0.0
[1mcollecting ... [0m[1m
collected 342 items / 8 deselected / 334 selected                              [0m

tests/test_app.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33m                                                [  2%][0m
tests/test_main.py [32m.[0m[33m                                                     [  2%][0m
tests/test_utils.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[33m                               [  8%][0m
tests/integration/test_normcap.py [32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0m[32m.[0mFatal Python error: Segmentation fault

Current thread 0x00007ffff7eb7740 (most recent call first):
  File "/nix/store/qkg49mhfmax01nmq5w0x5g1c7cjmjjcg-python3.11-pytest-qt-4.2.0/lib/python3.11/site-packages/pytestqt/qtbot.py", line 713 in mouseRelease
  File "/build/source/tests/conftest.py", line 241 in _select_region
  File "/build/source/tests/integration/test_normcap.py", line 26 in test_normcap_ocr_testcases
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/python.py", line 194 in pytest_pyfunc_call
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_callers.py", line 77 in _multicall
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_manager.py", line 115 in _hookexec
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_hooks.py", line 493 in __call__
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/python.py", line 1792 in runtest
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/runner.py", line 169 in pytest_runtest_call
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_callers.py", line 77 in _multicall
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_manager.py", line 115 in _hookexec
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_hooks.py", line 493 in __call__
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/runner.py", line 262 in <lambda>
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/runner.py", line 341 in from_call
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/runner.py", line 261 in call_runtest_hook
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/runner.py", line 222 in call_and_report
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/runner.py", line 133 in runtestprotocol
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/runner.py", line 114 in pytest_runtest_protocol
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_callers.py", line 77 in _multicall
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_manager.py", line 115 in _hookexec
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_hooks.py", line 493 in __call__
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/main.py", line 350 in pytest_runtestloop
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_callers.py", line 77 in _multicall
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_manager.py", line 115 in _hookexec
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_hooks.py", line 493 in __call__
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/main.py", line 325 in _main
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/main.py", line 271 in wrap_session
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/main.py", line 318 in pytest_cmdline_main
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_callers.py", line 77 in _multicall
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_manager.py", line 115 in _hookexec
  File "/nix/store/wbphc7m4dc2zyqc8fm584kxxi0qbplar-python3.11-pluggy-1.3.0/lib/python3.11/site-packages/pluggy/_hooks.py", line 493 in __call__
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/config/__init__.py", line 169 in main
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/_pytest/config/__init__.py", line 192 in console_main
  File "/nix/store/nkk6s4yh6lx94xdbkwvqygrm4dpm4rgr-python3.11-pytest-7.4.3/lib/python3.11/site-packages/pytest/__main__.py", line 5 in <module>
  File "<frozen runpy>", line 88 in _run_code
  File "<frozen runpy>", line 198 in _run_module_as_main

Extension modules: xxsubtype, shiboken6.Shiboken, PySide6.QtCore, PySide6.QtGui, PySide6.QtWidgets, PySide6.QtNetwork, PySide6.QtDBus, PySide6.QtTest (total: 8)
/nix/store/dy3zdx1gvhvc6vv73axj8axm209nby7v-pytest-check-hook/nix-support/setup-hook: line 53:   571 Segmentation fault      (core dumped) /nix/store/5k91mg4qjylxbfvrv748smfh51ppjq0g-python3-3.11.6/bin/python3.11 -m pytest -k "not test_wl_copy and not test_update_checker_triggers_checked_signal and not test_urls_reachable and not test_synchronized_capture" --ignore="tests/tests_gui/test_downloader.py" --ignore="tests/tests_gui/test_language_manager.py"
/nix/store/d4jf1cbbk494zwgbqz31pxgigpsbh6w2-stdenv-linux/setup: line 1553: pop_var_context: head of shell_variables not a function context
```

pbsds commented:
I'm unable to reproduce your failure on master, it may have been temporary?

I disabled the test `tests/integration/test_tray_menu.py` which is new in this bump. It tests darwin functionality which is unlikely available in the sandbox.

pbsds commented:
My push-and-wait-for-ofborg debugging loop has reached the limits of my patience, can anyone in @nixos/darwin-maintainers have a look?

Et7f3 commented:
> Well, that's weird. On _another_ machine I just got this build error and after checking it now doesn't build on the aforementioned machines anymore:
> 
> ```
> Extension modules: xxsubtype, shiboken6.Shiboken, PySide6.QtCore, PySide6.QtGui, PySide6.QtWidgets, PySide6.QtNetwork, PySide6.QtDBus, PySide6.QtTest (total: 8)
> /nix/store/dy3zdx1gvhvc6vv73axj8axm209nby7v-pytest-check-hook/nix-support/setup-hook: line 53:   571 Segmentation fault      (core dumped) /nix/store/5k91mg4qjylxbfvrv748smfh51ppjq0g-python3-3.11.6/bin/python3.11 -m pytest -k "not test_wl_copy and not test_update_checker_triggers_checked_signal and not test_urls_reachable and not test_synchronized_capture" --ignore="tests/tests_gui/test_downloader.py" --ignore="tests/tests_gui/test_language_manager.py"
> /nix/store/d4jf1cbbk494zwgbqz31pxgigpsbh6w2-stdenv-linux/setup: line 1553: pop_var_context: head of shell_variables not a function context
> ```

Check the RAM consumed by other app. Try to disable multi-threading it can does this. Or try to have more swap.

I

Et7f3 commented:
> My push-and-wait-for-ofborg debugging loop has reached the limits of my patience, can anyone in @ NixOS/darwin-maintainers have a look?
I don't understand why you ping darwin teams since the only failure are on linux.

stepbrobd commented:
Result of `nixpkgs-review pr 275063` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>normcap</li>
    <li>normcap.dist</li>
  </ul>
</details>

<details>
<summary>
'nix log /nix/store/6ywqxk646wqk5fgmfy1qa1k9ik97xaj4-normcap-0.5.2.drv'
</summary>

```console
$ nix log /nix/store/6ywqxk646wqk5fgmfy1qa1k9ik97xaj4-normcap-0.5.2.drv    
warning: The interpretation of store paths arguments ending in `.drv` recently changed. If this command is now failing try again with '/nix/store/6ywqxk646wqk5fgmfy1qa1k9ik97xaj4-normcap-0.5.2.drv^*'
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/8085wav5zkhj8zi4jnk7xakhpf57nncf-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tests/tests_screengrab/test_utils.py
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
* Getting build dependencies for wheel...
* Building wheel...
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.mo
68 of 68 messages (100%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.mo
68 of 68 messages (100%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.mo
Successfully built normcap-0.5.2-py3-none-any.whl
Finished creating a wheel...
/private/tmp/nix-build-normcap-0.5.2.drv-0/source/dist /private/tmp/nix-build-normcap-0.5.2.drv-0/source
Unpacking to: unpacked/normcap-0.5.2...OK
Repacking wheel as ./normcap-0.5.2-py3-none-any.whl...creating './normcap-0.5.2-py3-none-any.whl' and adding 'unpacked/normcap-0.5.2' to it
adding 'normcap/__init__.py'
adding 'normcap/__main__.py'
adding 'normcap/app.py'
adding 'normcap/utils.py'
adding 'normcap/clipboard/__init__.py'
adding 'normcap/clipboard/main.py'
adding 'normcap/clipboard/handlers/__init__.py'
adding 'normcap/clipboard/handlers/base.py'
adding 'normcap/clipboard/handlers/pbcopy.py'
adding 'normcap/clipboard/handlers/qtclipboard.py'
adding 'normcap/clipboard/handlers/windll.py'
adding 'normcap/clipboard/handlers/wlclipboard.py'
adding 'normcap/clipboard/handlers/xclip.py'
adding 'normcap/gui/__init__.py'
adding 'normcap/gui/constants.py'
adding 'normcap/gui/downloader.py'
adding 'normcap/gui/introduction.py'
adding 'normcap/gui/language_manager.py'
adding 'normcap/gui/loading_indicator.py'
adding 'normcap/gui/localization.py'
adding 'normcap/gui/menu_button.py'
adding 'normcap/gui/models.py'
adding 'normcap/gui/notification.py'
adding 'normcap/gui/resources.py'
adding 'normcap/gui/settings.py'
adding 'normcap/gui/system_info.py'
adding 'normcap/gui/tray.py'
adding 'normcap/gui/update_check.py'
adding 'normcap/gui/utils.py'
adding 'normcap/gui/window.py'
adding 'normcap/ocr/__init__.py'
adding 'normcap/ocr/enhance.py'
adding 'normcap/ocr/magic.py'
adding 'normcap/ocr/models.py'
adding 'normcap/ocr/recognize.py'
adding 'normcap/ocr/tesseract.py'
adding 'normcap/ocr/magics/__init__.py'
adding 'normcap/ocr/magics/base_magic.py'
adding 'normcap/ocr/magics/email_magic.py'
adding 'normcap/ocr/magics/multi_line_magic.py'
adding 'normcap/ocr/magics/paragraph_magic.py'
adding 'normcap/ocr/magics/single_line_magic.py'
adding 'normcap/ocr/magics/url_magic.py'
adding 'normcap/ocr/magics/url_utils.py'
adding 'normcap/resources/README.md'
adding 'normcap/resources/__init__.py'
adding 'normcap/resources/icons/normcap.png'
adding 'normcap/resources/icons/notification.png'
adding 'normcap/resources/icons/parse.png'
adding 'normcap/resources/icons/raw.png'
adding 'normcap/resources/icons/resources.qrc'
adding 'normcap/resources/icons/settings.png'
adding 'normcap/resources/icons/tray.png'
adding 'normcap/resources/icons/tray_done.png'
adding 'normcap/resources/icons/tray_processing.png'
adding 'normcap/resources/images/darwin-intro-1.png'
adding 'normcap/resources/images/darwin-intro-2.png'
adding 'normcap/resources/images/darwin-intro-3.png'
adding 'normcap/resources/images/darwin-intro-4.png'
adding 'normcap/resources/images/linux-intro-1.png'
adding 'normcap/resources/images/linux-intro-2.png'
adding 'normcap/resources/images/linux-intro-3.png'
adding 'normcap/resources/images/linux-intro-4.png'
adding 'normcap/resources/images/win32-intro-1.png'
adding 'normcap/resources/images/win32-intro-2.png'
adding 'normcap/resources/images/win32-intro-3.png'
adding 'normcap/resources/images/win32-intro-4.png'
adding 'normcap/resources/locales/README.md'
adding 'normcap/resources/locales/de_DE/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/es_ES/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/fr_FR/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/hi_IN/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/it_IT/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/pl_PL/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/pt_PT/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/ru_RU/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/sv_SE/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/uk_UA/LC_MESSAGES/messages.mo'
adding 'normcap/resources/locales/zh_CN/LC_MESSAGES/messages.mo'
adding 'normcap/resources/tessdata/.keep'
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
unpacking source archive /nix/store/8085wav5zkhj8zi4jnk7xakhpf57nncf-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tests/tests_screengrab/test_utils.py
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
Running phase: configurePhase
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
* Getting build dependencies for wheel...
* Building wheel...
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/ru_RU/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/es_ES/LC_MESSAGES/messages.mo
68 of 68 messages (100%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/de_DE/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/fr_FR/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pl_PL/LC_MESSAGES/messages.mo
68 of 68 messages (100%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/sv_SE/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/uk_UA/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/it_IT/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/hi_IN/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/zh_CN/LC_MESSAGES/messages.mo
6 of 68 messages (8%) translated in /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.po
compiling catalog /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.po to /private/tmp/nix-build-normcap-0.5.2.drv-0/source/normcap/resources/locales/pt_PT/LC_MESSAGES/messages.mo
Successfully built normcap-0.5.2-py3-none-any.whl
Finished creating a wheel...
/private/tmp/nix-build-normcap-0.5.2.drv-0/source/dist /private/tmp/nix-build-normcap-0.5.2.drv-0/source
Unpacking to: unpacked/normcap-0.5.2...OK
Repacking wheel as ./normcap-0.5.2-py3-none-any.whl...creating './normcap-0.5.2-py3-none-any.whl' and adding 'unpacked/normcap-0.5.2' to it
adding 'normcap/__init__.py'
adding 'normcap/__main__.py'
adding 'normcap/app.py'
adding 'normcap/utils.py'
adding 'normcap/clipboard/__init__.py'
adding 'normcap/clipboard/main.py'
adding 'normcap/clipboard/handlers/__init__.py'
adding 'normcap/clipboard/handlers/base.py'
adding 'normcap/clipboard/handlers/pbcopy.py'
...skipping...
adding 'normcap/resources/tessdata/LICENSE.txt'
adding 'normcap/resources/tessdata/README.txt'
adding 'normcap/screengrab/__init__.py'
adding 'normcap/screengrab/dbus_portal.py'
adding 'normcap/screengrab/dbus_shell.py'
adding 'normcap/screengrab/exceptions.py'
adding 'normcap/screengrab/grim.py'
adding 'normcap/screengrab/main.py'
adding 'normcap/screengrab/permissions.py'
adding 'normcap/screengrab/qt.py'
adding 'normcap/screengrab/utils.py'
adding 'normcap-0.5.2.dist-info/licenses/LICENSE'
adding 'normcap-0.5.2.dist-info/METADATA'
adding 'normcap-0.5.2.dist-info/WHEEL'
adding 'normcap-0.5.2.dist-info/entry_points.txt'
adding 'normcap-0.5.2.dist-info/RECORD'
OK
/private/tmp/nix-build-normcap-0.5.2.drv-0/source
Finished executing pypaBuildPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "installPhase" }
Running phase: installPhase
Executing pypaInstallPhase
Successfully installed normcap-0.5.2-py3-none-any.whl
Finished executing pypaInstallPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
Running phase: pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
Running phase: fixupPhase
checking for references to /private/tmp/nix-build-normcap-0.5.2.drv-0/ in /nix/store/y46hfvp407x53g7ffrgswmd0mgvpd2y7-normcap-0.5.2...
patching script interpreter paths in /nix/store/y46hfvp407x53g7ffrgswmd0mgvpd2y7-normcap-0.5.2
stripping (with command strip and flags -S) in  /nix/store/y46hfvp407x53g7ffrgswmd0mgvpd2y7-normcap-0.5.2/lib /nix/store/y46hfvp407x53g7ffrgswmd0mgvpd2y7-normcap-0.5.2/bin
checking for references to /private/tmp/nix-build-normcap-0.5.2.drv-0/ in /nix/store/88jiwhzmb6dzh33qnfy8as64aw5sc8yl-normcap-0.5.2-dist...
patching script interpreter paths in /nix/store/88jiwhzmb6dzh33qnfy8as64aw5sc8yl-normcap-0.5.2-dist
Rewriting #!/nix/store/z0nnwdh4p692yvf3qgwgj3bn78w99q6y-python3-3.11.6/bin/python3.11 to #!/nix/store/z0nnwdh4p692yvf3qgwgj3bn78w99q6y-python3-3.11.6
wrapping `/nix/store/y46hfvp407x53g7ffrgswmd0mgvpd2y7-normcap-0.5.2/bin/normcap'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
@nix { "action": "setPhase", "phase": "installCheckPhase" }
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Running phase: pythonImportsCheckPhase
Executing pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Running phase: pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform darwin -- Python 3.11.6, pytest-7.4.3, pluggy-1.3.0
PySide6 6.6.0 -- Qt runtime 6.6.1 -- Qt compiled 6.6.1
rootdir: /private/tmp/nix-build-normcap-0.5.2.drv-0/source
configfile: pyproject.toml
testpaths: tests
plugins: qt-4.3.1
collecting ... ^Mcollected 321 items / 15 deselected / 306 selected

tests/test_app.py ...../nix/store/5hrvnvlb7bswkyy5v4k6k49xq42hkg12-pytest-check-hook/nix-support/setup-hook: line 53: 78338 Trace/BPT trap: 5       /nix/store/z0nnwdh4p692yvf3qgwgj3bn78w99q6y-python3-3.11.6/bin/python3.11 -m pytest -k "not test_wl>
/nix/store/sf52i9wcklk5i5f2w15p0kng8dq3qqwx-stdenv-darwin/setup: line 1562: pop_var_context: head of shell_variables not a function context
```
</details>

pbsds commented:
>  I don't understand why you ping darwin teams since the only failure are on linux.

Huh, it builds locally on linux, but the ofborg error on linux is the same I kept seeing on darwin and the same as posted by `@stepbrobd`.

pbsds committed:
normcap: 0.5.2 -> 0.5.4

release: https://github.com/dynobo/normcap/releases/tag/v0.5.4

diff: https://github.com/dynobo/normcap/compare/v0.5.2...v0.5.4

pbsds committed:
normcap: mark broken on darwin

nadir-ishiguro commented:
Thanks for the update. I got the following build error:

```
Checking runtime dependencies for normcap-0.5.4-py3-none-any.whl
  - pyside6-essentials not installed
```

Rest of the log:
https://gist.github.com/nadir-ishiguro/e04273d85ac9c20a388b59c78b96e5e5

pbsds commented:
And now aarch64-linux is broken. Any ideas?

nadir-ishiguro commented:
> And now aarch64-linux is broken. Any ideas?

Sorry, I don't. I'm just interested in the update and failed to fix the derivation myself before seeing your PR.

pbsds commented:
ok, so after the re-eval it seems that the `checkPhase` is flaky, since this time around the x86_64-linux version failed instead of aarch64-linux...

pbsds committed:
normcap: disable flaky test
